app/timetables/models.py

```diff
@@ -1,3 +1,16 @@
 from __future__ import unicode_literals
 
 from django.db import models
+
+from app.shared.models import CommonInfo
+
+
+class Timetable(CommonInfo):
+    name = models.TextField()
+    code = models.TextField()
+    api_key = models.TextField()
```

Comment:
If we are not implementing these fields as textarea form inputs, then let's use CharField instead and specify a max length while at it. A max length helps add a layer of security by restricting the acceptable input to a certain length. We could use a max length of 60 characters (`max_length=60`)